packages/dso-toolkit/src/components/list/list.stories-of.ts

```diff
@@ -93,6 +93,31 @@ export function storiesOfList<Implementation, Templates, TemplateFnReturnType>(
       },
     });
 
+    stories.add("icons", template, {
+      args: {
+        type: Type.Ul,
+        items: [
+          {
+            text: "Cras justo odio",
+          },
+          {
+            text: "Dapibus ac facilisis in",
+            modifier: "forbidden",
+          },
+          {
+            text: "Morbi leo risus",
+          },
+          {
+            text: "Porta ac consectetur ac",
+            modifier: "status-warning",
+          },
+          {
+            text: "Vestibulum at eros",
```

Comment:
Kunnen we iets met echte content doen? In het voorbeeld van Martin staat volgens mij een prima lijstje wat we kunnen overnemen. In principe willen we geen Lorem teksten meer.

Reply:
Ik zie geen lijstje of voorbeeld

Reply:
Ik ook niet :(